assets/src/admin/paired-browsing/app.js

```diff
@@ -0,0 +1,189 @@
+/* global ampSlug, ampPairedBrowsingQueryVar */
```

Comment:
If deconstructing these variables from `window`, neither such a `global` definition nor the change to the ESLint config should not be needed:

```js
const { ampSlug, ampPairedBrowsingQueryVar, alert, history, parent } = window;
```

Reply:
Yep that's a better solution, thanks :100: